lib/components/DebugInfo.tsx

```diff
@@ -39,10 +38,6 @@ export default function DebugInfo() {
 					'unknown'
 				}`}
 				{'\n'}
-				{`build ${Constants.expoConfig?.version ?? 'unknown'}-${(
-					Updates.createdAt ?? new Date()
-				).toISOString()}`}
```

Comment:
this is very useful to have when trying to figure out which version an issue is happening on, I would keep this. It works even without an EAS Updates setup, and in development mode.

Reply:
Added back the version, but without the created at to avoid having `expo-updates` just for this